factcast-store-test/src/main/java/org/factcast/store/test/AbstractFactStoreTest.java

```diff
@@ -572,23 +559,23 @@ protected void testSerialOf() {
 
     @Test
     protected void testChecksMandatoryNamespaceOnPublish() {
-        Assertions.assertThrows(IllegalArgumentException.class, () -> uut.publish(Fact.of(
-                "{\"id\":\"" + UUID.randomUUID() + "\",\"type\":\"someType\"}",
-                "{}")));
+        Assertions.assertThrows(IllegalArgumentException.class,
+                () -> uut.publish(Fact.of("{\"id\":\"" + UUID.randomUUID()
+                        + "\",\"type\":\"someType\"}", "{}")));
     }
 
     @Test
     protected void testChecksMandatoryIdOnPublish() {
-        Assertions.assertThrows(IllegalArgumentException.class, () -> uut.publish(Fact.of(
-                "{\"ns\":\"default\",\"type\":\"someType\"}", "{}")));
+        Assertions.assertThrows(IllegalArgumentException.class,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assertions.assertThrows' due to existing static import 'org.junit.jupiter.api.Assertions.assertThrows'](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4877539)